src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] data = command.split(" ");
+                String name = data[0];
+                int age = Integer.parseInt(data[1]);
+                int value = Integer.parseInt(data[2]);
+                double risk = Double.parseDouble(data[3]);
+                user = new User(name, age);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException | IndexOutOfBoundsException e) {
+                System.out.println("Please, input correct data");
+                handle();
+            }
+            betDao.add(bet);
+            userDao.add(user);
+            userDao.checkAge(user);
+            System.out.println(bet == null ? null : user.toString() + " " + bet.toString());
```

Comment:
what if user is null?

Reply:
if our bet == null then our user automaticly == null and vice versa so we need only one check